client/src/models/InteractedContact.ts

```diff
@@ -27,6 +27,13 @@ interface InteractedContact extends Person {
     involvementReason: number | null;
     involvedContactId: number | null;
     placeName?: string;
-};
+    finalEpidemiologicalStatusDesc?: string;
+    ColorCode?: string;
```

Comment:
please change to small c

Reply:
👍

Reply:
It's supposed to be colorCode like the others...